### DISCUSSION TITLE
Catchup for stream stalled

### DISCUSSION THREAD

**Discussion by BinaryArchaism** [nats-io/nats-server#4064] [Q&A]
Hello guys! Could you please help me.
I have an error on my node of 3 node nats jetstream enabled cluster: `Catchup for stream '...' stalled`.
How can I fix this? I tried to remove all node files and start it back to the cluster, but it didn't help. 
Maybe it is only connectivity issue between nodes? 
The origin of this error: [here](https://github.com/nats-io/nats-server/blob/main/server/jetstream_cluster.go#L7587) or [here](https://github.com/nats-io/nats-server/blob/main/server/jetstream_cluster.go#L8136)

Nats-server v2.9.15
Stream storage type: memory

**Comment by derekcollison** [comment]
Move this to an issue and share information on system topology and setup, and share `nats stream info` and as a system user `nats server ls` and `nats server report jetstream`

**Comment by BinaryArchaism** [comment]
Not relevant anymore. With latest all is ok
